src/utils/chartUtils.ts

```diff
@@ -0,0 +1,7 @@
+//TODO: Some method to generate the palette of colors
+//we want for the graph lines.
+export const getChartLineColors = (index: number): string => {
+  const defaultHue = 66;
+  const multiplier = 4;
+  return `hsl(${defaultHue + index * multiplier}deg 82% 55%)`;
+};
```

Comment:
Worried about how it will look when we have 40 series in one graph. So far I applied a very simple logic that slightly varies the hue with each series, so the combination of colors should be nice to the eye. 
TBD.